src/components-styled/content-header/metadata.tsx

```diff
@@ -0,0 +1,161 @@
+
+export function Metadata(props: MetadataProps) {
+  const { dataSources, datumsText, dateInfo, dateOfInsertionUnix } = props;
+
+  let dateText = '';
+  if (typeof dateInfo === 'number') {
+    const dateOfReport = formatDateFromSeconds(dateInfo, 'weekday-medium');
+    const dateOfInsertion = formatDateFromSeconds(
+      dateOfInsertionUnix,
+      'weekday-medium'
+    );
+    dateText = replaceVariablesInText(datumsText, {
+      dateOfReport,
+      dateOfInsertion,
+    });
+  } else {
+    const weekStart = formatDateFromSeconds(
+      dateInfo.weekStartUnix,
+      'weekday-medium'
+    );
+    const weekEnd = formatDateFromSeconds(
+      dateInfo.weekEndUnix,
+      'weekday-medium'
+    );
+    const dateOfInsertion = formatDateFromSeconds(
+      dateOfInsertionUnix,
+      'weekday-medium'
+    );
+    dateText = replaceVariablesInText(datumsText, {
+      weekStart,
+      weekEnd,
+      dateOfInsertion,
+    });
+  }
+
+  const dataDownloads = dataSources
+    .filter((x) => Boolean(x.download.trim()))
+    .map((x) => ({ href: x.download, text: x.text }));
+
+  return (
+    <Box>
+      <Box
+        display="flex"
+        alignItems="flex-start"
+        color="annotation"
+        my={3}
+        mx={0}
+      >
+        <Box as="span" minWidth="1.8rem" mt={1}>
+          <ClockIcon aria-hidden color={theme.colors.annotation} />
+        </Box>
+        <Text margin={0}>{dateText}</Text>
+      </Box>
+
+      <MetadataItem
+        icon={<DatabaseIcon aria-hidden />}
+        items={dataSources}
+        label={text.source}
+      />
+
+      <MetadataItem
+        icon={
+          <DownloadIcon
+            aria-hidden
+            color={theme.colors.annotation}
+            width="1em"
+            height="1em"
+          />
+        }
+        items={dataDownloads}
+        label={text.download}
+      />
+    </Box>
+  );
+}
+
+interface MetadataItemProps {
+  icon: JSX.Element;
+  label: string;
+  items: {
+    href: string;
+    text: string;
+  }[];
+}
+
+function MetadataItem(props: MetadataItemProps) {
+  const { icon, label, items } = props;
+
+  if (!items.length) {
+    return null;
+  }
+
+  return (
+    <Box
+      display="flex"
+      alignItems="flex-start"
+      color="annotation"
+      my={3}
+      mx={0}
+    >
+      <Box as="span" minWidth="1.8rem" mt={1}>
+        {icon}
+      </Box>
+      <Text margin={0}>
+        {label}:{' '}
+        {items.map((item, index) => {
+          if (index) {
+            return (
+              <>
+                {' '}
+                &amp;{' '}
+                <ExternalLink
+                  href={item.href}
+                  text={item.text}
+                  key={`label_${index}_${item.href}`}
+                />
+              </>
+            );
+          }
+          return (
+            <ExternalLink
+              href={item.href}
+              text={item.text}
+              key={`label_${index}_${item.href}`}
```

Comment:
should be sufficient:
```suggestion
              key={index + item.href}
```